lib/opencensus_phoenix/telemetry.ex

```diff
@@ -99,7 +77,46 @@ defmodule OpencensusPhoenix.Telemetry do
     })
   end
 
-  def handle_event([:phoenix, :live_component, :handle_event, :stop], meas, meta, _config) do
+  def handle_event([:phoenix, :live_component, :handle_event, :stop], meas, meta, _) do
+    :ocp.finish_span()
+  end
+
+  def handle_event([:phoenix, :router_dispatch, :start], measurements, meta, _) do
+    route_info =
+      case meta do
+        %{plug: Phoenix.LiveView.Plug, phoenix_live_view: {module, action}} ->
+          "Elixir." <> view = to_string(module)
+          %{module: view, action: action, view: view, root_view: view}
+
+        %{plug: module, plug_opts: action} ->
+          "Elixir." <> module = to_string(module)
+          %{module: module, action: action}
+      end
+
+    action =
+      case route_info.action do
+        [schema: schema, json_codec: _] -> schema
```

Comment:
This can fail badly if the order of the information will be different than the presented (keyword lists matching is order-dependant. Additionally it can cause problems if there is more elements than 2 in this list.